packages/build/bin/Utils.js

```diff
@@ -3,24 +3,25 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.loadJson = exports.overlay = exports.merge = exports.isDefined = exports.requireNotNull = void 0;
+exports.requireNotNull = requireNotNull;
```

Comment:
Anything in this bin folder is derived from the Typescript sources and doesn't need to be edited manually as the next build will overwrite it.

Reply:
should not be part of scm then...removed